R/mod_load_data.R

```diff
@@ -32,6 +32,14 @@ mod_load_data_server <- function(input, output, session, cons){
                                        project_crs = get_golem_config("project_crs"))
     message("Finished loading trips")
 
+    if (getOption("emdash.anon_locations"))  {
+        message("About to anonymize trajectories")
+        data_r$anon_trips <- data_r$trips %>% anonymize_uuid
+        message("Finished anonymizing trajectories")
+    } else {
+        data_r$anon_trips <- data_r$trips
+    }
+
```

Comment:
Let's not create another variable here. Just assign anonymised trips to `data_r$trips`.

Reply:
I created the other variable because we don't want to hide the UUIDs on the trips and participants tables. Only on the geospatial data.

Reply:
Since the number of trips and their time is not very identifying.

Reply:
Is this better? https://github.com/asiripanich/emdash/pull/11/commits/f6d5163f5cf9fcd71c88ba5c8ac3308fdc4eb31d

For large databases I don't think it is a good idea to replicate the table just for that purpose.